server/migrations/1723187773982_user-group-refactoring.ts

```diff
@@ -0,0 +1,50 @@
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.sql(
+    `
+      ALTER TABLE APPLICATION.user RENAME groups TO organizations;
+      ALTER TABLE DATA.files ALTER COLUMN groups TYPE TEXT;
+      ALTER TABLE DATA.files RENAME groups TO organization;
+      ALTER TABLE DATA.files DROP CONSTRAINT pk_files CASCADE;
+      ALTER TABLE DATA.files
```

Comment:
I guess it would complicate the migration quite much, but would it be possible to convert the existing data into the new single-column format here before dropping the columns and losing that information? All existing surveys that refer to any files would also fail after this migration.

I'm also fairly certain that when we add a primary key constraint to a new column that only has NULL values, the migration would fail.